src/components/RP_GettingStarted.component

```diff
@@ -0,0 +1,82 @@
+
+            <h2 class="slds-text-heading_medium slds-align_absolute-center">{!$Label.RP_TrailheadSubtitle}</h2>
+            <p class="slds-m-vertical_large">{!$Label.RP_TrailheadParagraph}</p>
+
+            <apex:outputLink target="_blank" styleClass="slds-align_absolute-center links" rendered="{!NOT(ISNULL(linksData.trailheadLink))}" value="{!linksData.trailheadLink}"> {!$Label.RP_TrailheadLinkLabel} </apex:outputLink>
+
+        </div>
+
+        <div id="bordersDiv" class="slds-p-horizontal_xx-large slds-border_right slds-border_left slds-size--1-of-1 slds-large-size--1-of-3">
+
+            <img class="slds-col slds-p-vertical_medium slds-align_absolute-center" id="usImage" src="{!URLFOR($Resource.RP_Resources, 'Resources/images/Image_US.png' )}" width="175" height="60"/>
+
+            <h2 class="slds-col slds-text-heading_medium slds-align_absolute-center">{!$Label.RP_USSubtitle}</h2>
+            <p class="slds-m-vertical_large">{!$Label.RP_USParagraph}</p>
+
+            <apex:outputLink target="_blank" styleClass="slds-align_absolute-center links" rendered="{!NOT(ISNULL(linksData.usLink))}" value="{!linksData.usLink}"> {!$Label.RP_UsHubLinkLabel} </apex:outputLink>
+
+            <apex:outputLink target="_blank" styleClass="slds-align_absolute-center links" rendered="{!NOT(ISNULL(linksData.salesforceOrgLink))}" value="{!linksData.salesforceOrgLink}"> {!$Label.RP_SalesforceOrgLinkLabel} </apex:outputLink>
+
+        </div>
+
+        <div class="slds-p-horizontal_xx-large slds-size--1-of-1 slds-large-size--1-of-3">
+
+            <img class="slds-p-vertical_medium slds-align_absolute-center" id="diveDeeperImage" src="{!URLFOR($Resource.RP_Resources, 'Resources/images/Image_DiveDeeper.png' )}" width="130" height="60"/>
+
+            <h2 class="slds-text-heading_medium slds-align_absolute-center">{!$Label.RP_DeeperSubtitle}</h2>
+            <p class="slds-m-vertical_large">{!$Label.RP_DeeperParagraph}</p>
+
+            <apex:outputLink target="_blank" styleClass="slds-align_absolute-center links" rendered="{!NOT(ISNULL(linksData.nspsLink))}" value="{!linksData.nspsLink}"> {!$Label.RP_NpspLinkLabel} </apex:outputLink>
+
+            <apex:outputLink target="_blank" styleClass="slds-align_absolute-center links" rendered="{!NOT(ISNULL(linksData.webinarLink))}" value="{!linksData.webinarLink}"> {!$Label.RP_WebinarLinkLabel} </apex:outputLink>
+
+            <apex:outputLink target="_blank" styleClass="slds-align_absolute-center links" rendered="{!NOT(ISNULL(linksData.customerJourneyLink))}" value="{!linksData.customerJourneyLink}"> {!$Label.RP_CustomerJourneyLinkLabel} </apex:outputLink>
+
+
+        </div>
+
+    </div>
+
+    <img id="mountainsImage" src="{!URLFOR($Resource.RP_Resources, 'Resources/images/Image_Mountains.png' )}" width="2000" height="80" />
+
+    <script src="{!URLFOR($Resource.RP_Resources, 'Resources/js/jquery-3.2.1.min.js' )}" />
```

Comment:
So we'd prefer avoiding yet another version of jquery being included if possible.  If you look in our current CumulusStaticResources.resource (and the source under StaticResourceSources), you will see that we already have jquery-1-10.2.min.js and jquery-2.2.4.js.  If your usage is limited enough that one of these existing versions works for you, please use it.  Otherwise, include 3.2.1.min.js in our static resource.  This helps encourage sharing and avoiding lots of different versions floating around.

My next question is about using the <script> tag to include it rather than <apex:includeScript>.  If you use <apex:includeScript> Multiple references to the same script are de-duplicated, which seems like a win, since all three of your visualforce components include jquery.